lib/Echidna/Test.hs

```diff
@@ -51,11 +53,11 @@ classifyRes _ = ResOther
 
 -- | Given a 'SolTest', evaluate it and see if it currently passes.
 checkETest :: (MonadReader x m, Has TestConf x, Has TxConf x, MonadState y m, Has VM y, MonadThrow m)
-           => SolTest -> m Bool
-checkETest t = do
+           => EventMap -> SolTest -> m Bool
+checkETest em t = do
   TestConf p s <- asks getter
   g <- view (hasLens . propGas)
-  vm <- get -- save EVM state
+  st <- get -- save EVM state
```

Comment:
we should be consistent here. what's wrong with `vm`? also historically this has been `og` lol.

Reply:
Fixed.